api/web_service.py

```diff
@@ -0,0 +1,36 @@
+import threading
+from flask import Flask
+from flask_cors import CORS
+from src.interfaces import IAccount
+
+class WebService():
+
+    account: IAccount
+
+    def __init__(self, name, account: IAccount) -> None:
+        
```

Comment:
it looks like this was updated to accept account by dependency injection, thanks